common/pkg/config/containers.conf

```diff
@@ -317,11 +317,13 @@ default_sysctls = [
 #
 #umask = "0022"
 
-# Default way to to create a User namespace for the container
+# Default way to create a USER namespace for the container.
 # Options are:
-# `auto`        Create unique User Namespace for the container.
-# `host`    Share host User Namespace with the container.
-#
+#   `private` Create private USER Namespace for the container.
```

Comment:
this also needs the `without adding any UID mappings.` comment to keep them both in sync

Reply:
Good catch, that should be fixed now.